integration-tests/src/dust.rs

```diff
@@ -517,6 +517,54 @@ mod atoken_dust {
 	}
 }
 
+#[test]
+fn is_whitelisted_runtime_api_should_not_work_for_dustable_accounts() {
+	use pallet_duster::runtime_decl_for_duster_api::DusterApi;
```

Comment:
This is duplicated in every test. Can be moved to the top of the file, where all imports are.